debian/changelog

```diff
@@ -1,11 +1,13 @@
-cloud-init (20.1-10-g71af48df-0ubuntu2) UNRELEASED; urgency=medium
+cloud-init (20.1-10-g71af48df-0ubuntu3) focal; urgency=medium
```

Comment:
This version was never released, so I don't think we need to increment here?

Reply:
ah right good point.